test/integration/targets/var_reserved/tasks/task_vars_used.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: localhost
+  gather_facts: false
+  tasks:
+  - name: task fails due to overriding q, but we should also see warning
+    debug:
+      msg:  "{{q('pipe', 'pwd'}}"
```

Comment:
There is a templating issue here that is potentially not testing that `q` is not overridden.

It's missing a `)` before the `}}`

Reply:
weird, thought linting would have caught that, fixing

Reply:
#84563